### ISSUE TITLE
eti2zmq not created

### ISSUE THREAD

**Issue by andimik** [piratfm/eti-tools#37]
Using the latest version, I've enabled Zero MQ

```
CFLAGS+= -DHAVE_ZMQ
LDFLAGS+= -lzmq
```
but `eti2zmq` is not created, and the file `eti2zmq.o` is missing after `make`.

**Then I went back to 4eb5828ae915e8aa789642d7f34333c23693204d (14 March 2019) and here it works.**

Seems that on the following merge (files from 25 March 2018 = one year before) in e40d87f0b3fec7c0dfeb2ffdb0400a1b44ee7feb or bc13d09 this got lost.

Outputs:

[make_15march2019.txt](https://github.com/piratfm/eti-tools/files/4433187/make_15march2019.txt)
[make_14march2019.txt](https://github.com/piratfm/eti-tools/files/4433188/make_14march2019.txt)

I'm NOT a programmer, but it seems that in the `Makefile` the line should be

```
all: cleanapps ni2out ts2na na2ts na2ni edi2eti fedi2eti mpe2aac mpe2mpa mpe2ts dvb-ip-mpe2ts eti2zmq
```

and 

```
	install -m 755 eti2zmq $(DESTDIR)/usr/bin
```
at the end

@athoik could you check?

**Comment by athoik** [comment]
Hi, 

I think eti2zmq is missing in this line: https://github.com/piratfm/eti-tools/blob/ea9fc7e1fac21df749fcf2358dfa4ef58e7c4cd9/Makefile#L34

**Comment by andimik** [comment]
fixed in 8af78c2
